packages/luna/src/components/Lessons/UpcomingLessonsSummary.tsx

```diff
@@ -96,13 +120,34 @@ export const UpcomingLessonsSummary: React.FC<Props> = ({
                     >
                       {dayjs(lesson.start).format("dddd - D MMMM YYYY")}
                     </Typography>
-                    <Typography
-                      element="tiny-text"
-                      weight="regular"
-                      className="tw-text-natural-600"
-                    >
-                      {lesson.tutorName}
-                    </Typography>
+                    <div className="tw-flex tw-justify-between">
+                      <Tooltip
+                        content={
+                          <Typography className="tw-text-natuarl-950">
+                            {lesson.name}
+                          </Typography>
+                        }
+                      >
+                        <div>
+                          <Typography
+                            element="tiny-text"
+                            weight="regular"
+                            className="tw-block tw-text-natural-600 tw-me-1"
+                          >
+                            {lesson.name?.split(" ").slice(0, 2).join(" ")}
```

Comment:
This is not the right solution to fix the issue when the content is to long. Please truncate the text using css and don't slice it.